components/04-views/login/login.hbs

```diff
@@ -0,0 +1,90 @@
+                     login__tab-container
+                 "
+            >
+                <div class="
+                         tab__title {{#if tabs.registeredCustomer.active}}tab__title--active{{/if}}
+                         login__tab-title
+                "
+                     data-tab="{{ tabs.registeredCustomer.tabId }}"
+                >
+                    {{ tabs.registeredCustomer.title }}
+                </div>
+                <div class="
+                        tab__content login__tab-content {{#if tabs.registeredCustomer.active}}tab__content--active{{/if}}
+                        login__tab-content
+                     "
+                     data-content="{{ tabs.registeredCustomer.tabId }}"
+                >
+                    <div class="login__info">
+                        {{{ signInInfo }}}
+                    </div>
+
+                    <fieldset class="fieldset login__fieldset {{ signInSection.fieldset.class }}"
+                              {{{ signInSection.fieldset.attributes }}}
+                    >
+                        <legend class="fieldset__legends login__legend {{ signInSection.legend.class }}"
+                            {{{ signInSection.legend.attributes }}}
+                        >
+                            {{ signInSection.legend.text }}
+                        </legend>
+
+                        {{ render '@input' email }}
+                        {{ render '@input' password }}
+
+                    </fieldset>
+
+                    <div class="login__actions">
+                        {{ render '@button' loginButton }}
+                        <a href="{{ forgetPassword.href }}" class="login__forgot-password">
+                            {{ forgetPassword.text }}
+                        </a>
+                    </div>
+                    <div class="login__validation-tips">
+                        {{{ validationTips }}}
+                    </div>
+                </div>
+
+                <div class="
+                         tab__title {{#if tabs.newCustomer.active}}tab__title--active{{/if}}
+                         login__tab-title
+                     "
+                     data-tab="{{ tabs.newCustomer.tabId }}"
+                >
+                    {{ tabs.newCustomer.title }}
+                </div>
+                <div class="
+                         tab__content {{#if tabs.newCustomer.active}}tab__content--active{{/if}}
+                         login__tab-content
+                     "
+                     data-content="{{ tabs.newCustomer.tabId }}"
+                >
+                    <div class="login__info">
+                         {{{ createAccountInfo }}}
+                    </div>
+                    {{ render '@button' createAccountButton }}
+                </div>
+            </div>
+        </div>
+
+        <div class="login__sidebar">
+            {{ render '@image--picture' banner }}
+        </div>
```

Comment:
I see that this div is only difference between this variant and default version. Maybe we can use a some conditional instead of creating a whole new file ?

Reply:
imho easier for checking as variants.